spec/models/password_recovery_form_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+#:code, :email, :password, :password_confirmation
```

Comment:
This needed?

Reply:
Nope... shall remove it... It was there to make sure I tested everything